tests/validate/common/includes/cmdparser/cmdlineparser.h

```diff
@@ -110,17 +91,18 @@ class CmdLineParser {
    */
   CmdSwitch *getCmdSwitch(const char *key);
 
-  bool token_to_fullkeyname(const string &token, string &fullkey);
+  bool token_to_fullkeyname(const std::string &token, std::string &fullkey);
```

Comment:
Little awkward to have the function pass its output in fullkey rather than return the full key.  I understand that bool return value indicates if the fullkey was there, but couldn't an empty string do the same?

You may also want to comment the API.